src/routes/docs/widget/roadmap/+page.md

```diff
@@ -1,37 +1,40 @@
-# Future APIs (not implemented)
+# Future Support (not yet implemented)
 
-## Currently **unsupported**
+## Planned for version 1.1
 
 1. WebAuthn
-2. Push Authentication
-3. Recaptcha
-4. QR Code display
-5. TextOutputCallback with scripts
-6. Device Profile
-7. Email Suspend (Forgot Password/Username flows)
-8. Social Login
-9. Central Login
-10. SAML
-
-## Widget customization (future)
+2. Device Profile
+
+## Planned for version 1.2+
+
+1. Push Authentication
+2. ReCAPTCHA
+3. QR Code display
+4. TextOutputCallback with scripts
+5. Central Login
+6. SAML
+
+## Proposed future Widget APIs
+
+### Modal customization
 
 ```js
 new Widget({
   // ... previous config properties ...
 
   // All optional; default value is assigned below
-  customization: {
-    labels: 'floating', // "floating" or "stacked"
-    modalBackdrop: true, // boolean; display modal backdrop
-    modalAutoClose: true, // boolean; automatically close modal on success
+  style: {
+    modal: {
+      backdrop: true, // boolean; display modal backdrop
+    },
   },
 });
 ```
 
-## Additional modal events (future)
+### Multiple widget instances
 
-```js
-modal.onClose((event) => {
-  /* anything you want */
-});
-```
+Currently, the Login Widget supports only a single instance. There is a proposal to support multiple instances in the future. Let us know if this interests you.
```

Comment:
How would we suggest someone let us know? Zendesk? Perhaps a hyperlink to a zendesk ticket would be beneficial? Github issue?

Reply:
Honestly, I don't know at this time. Most of those who read this will be internal to ForgeRock, and will comment within the Slack channel. Eventually, it will be Backstage, likely.